warehouse/redshift/redshift.go

```diff
@@ -534,9 +523,19 @@ func Connect(cred RedshiftCredentialsT) (*sql.DB, error) {
 }
 
 func (rs *HandleT) dropDanglingStagingTables() bool {
-	sqlStatement := fmt.Sprintf(`select table_name
-								 from information_schema.tables
-								 where table_schema = '%s' AND table_name like '%s';`, rs.Namespace, fmt.Sprintf("%s%s", stagingTablePrefix, "%"))
+	sqlStatement := fmt.Sprintf(`
+		select
+		  table_name
+		from
+		  information_schema.tables
+		where
+		  table_schema = '%s'
+		  AND table_name like '%s%s';
+	`,
+		rs.Namespace,
+		warehouseutils.StagingTablePrefix(provider),
+		"%",
```

Comment:
You can use `%%` if you want a literal `%` https://pkg.go.dev/fmt

I find this a bit more readable, although I would prefer using prepared statements here.
```suggestion
		  AND table_name like '%s%%';
	`,
		rs.Namespace,
		warehouseutils.StagingTablePrefix(provider),
```

Reply:
Will make the changes.

Reply:
Done.